packages/website/pages/bids.tsx

```diff
@@ -17,6 +17,7 @@ import { contractAddress } from "../utils/config";
 import { alertMessageState, indexLoaderState } from "../state/recoil";
 
 const CONTRACT_REMOVE_BID_GAS = utils.format.parseNearAmount(`0.00000000020`); // 200 Tgas
+const YOCTO_NEAR = utils.format.parseNearAmount(`0.000000000000000000000001`);
```

Comment:
Isn't it better to use `formatNearAmount` here? https://github.com/near/near-api-js/blob/27ca6cddd12e62181fd37b72092de614788ff7c3/lib/utils/format.js#L30

Reply:
`formatNearAmount(1)` :)